lib/pipeline.js

```diff
@@ -342,17 +343,29 @@ class PipelineModel extends BaseModel {
                 // Get next jobs for when startFrom is ~pr
                 const nextJobs = workflowParser.getNextJobs(parsedConfig.workflowGraph, {
                     trigger: '~pr',
-                    prNum
+                    prNum,
+                    prChain: this.prChain
                 });
 
                 // Get all the missing PR- job names
-                const existingPRJobNames = prJobs.map(p => p.name);
-                const missingPRJobNames = nextJobs.filter(j => !existingPRJobNames.includes(j));
-
-                // Get the job name part, e.g. main from PR-1:main to create job
-                const jobsToCreate = missingPRJobNames.map(name => name.split(':')[1]);
-                const jobsToArchive = prJobs.filter(p => !nextJobs.includes(p.name));
-                const jobsToUnarchive = prJobs.filter(p => nextJobs.includes(p.name));
+                const existingPRJobNames = prJobs.map(prJob => prJob.name);
+                const missingPRJobNames =
+                    nextJobs.filter(nextJob => !existingPRJobNames.includes(nextJob));
+                let jobsToCreate;
+                let jobsToArchive;
+                let jobsToUnarchive;
+
+                winston.info(`prChain flag is ${this.prChain}.`);
```

Comment:
we should add more information for logging like pipeline ID

Reply:
I added pipelineId to this info log.